Neos.Neos/Classes/Domain/Service/SiteServiceInternals.php

```diff
@@ -60,6 +60,10 @@ public function removeSiteNode(SiteNodeName $siteNodeName): void
             $sitesNodeAggregate = $contentGraph->findRootNodeAggregateByType(
                 NodeTypeNameFactory::forSites()
             );
+            if (!$sitesNodeAggregate) {
+                // nothing to prune, we could probably also return here directly?
```

Comment:
That would be the current behavior with the exception I guess? Or rather we might actually throw here to tell people that this site does not exist?

Reply:
yes we throw currently. But as we want to remove a child node either way it seems fine to say, well we cant remove a thing here.

Reply:
I think, skipping is the better behavior.
But maybe we could throw at the end of this method if no node was actually removed?

Reply:
I dont think so. But with https://github.com/neos/neos-development-collection/issues/4470 we can definitely discuss how the full behaviour should be and adjust edge cases.